node/pod.go

```diff
@@ -185,6 +185,9 @@ func (pc *PodController) updatePodStatus(ctx context.Context, podFromKubernetes
 	kPod.Lock()
 	podFromProvider := kPod.lastPodStatusReceivedFromProvider.DeepCopy()
 	kPod.Unlock()
+	if cmp.Equal(podFromKubernetes.Status, podFromProvider.Status) {
```

Comment:
Stupid question, with the UpdatePod callback suppression, where would this get picked up?

Reply:
Sorry, I do nor get it. But `UpdatePod` would only update pod from `api-server` to `provider`.  This commit only change the logic in func `updatePodStatus`.

Reply:
I'm curious, what's the case where the provider would send duplicate updates?

Reply:
restart virtual-kubelet